<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShutdownDialog</class>
 <widget class="QDialog" name="ShutdownDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shutdown</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="timingLabel">
         <property name="text">
          <string>Timing:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="centerLayout">
       <item>
        <widget class="QComboBox" name="typeComboBox"/>
       </item>
       <item>
        <widget class="QComboBox" name="timingComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="binaryButtonBox">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEdit"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="addToPresetsCheckBox">
     <property name="text">
      <string>Add to &amp;Presets</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTimeEdit" name="timeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>D/M/Y H:M AP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>binaryButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShutdownDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>binaryButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShutdownDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
